client/state/selectors/index.js

```diff
@@ -60,6 +61,7 @@ export isDeactivatingJetpackModule from './is-deactivating-jetpack-module';
 export isDomainOnlySite from './is-domain-only-site';
 export isFetchingJetpackModules from './is-fetching-jetpack-modules';
 export isJetpackModuleActive from './is-jetpack-module-active';
+export isJetpackModuleUnavailableInDevelopmentMode from './is-jetpack-module-unavailable-in-development-mode';
```

Comment:
What's the use case where the selector  `isJetpackModuleUnavailableInDevelopmentMode` would be more practical than `isJetpackModuleAvailableInDevelopmentMode` ?